pages/manage/productions/index.test.js

```diff
@@ -0,0 +1,58 @@
+import Index from './Index';
+import Vue from 'vue';
+
+import { mountWithEnvironment } from '../../../test/mountWithEnvironment';
+
+test('returns full list when entering the page', async () => {
+  const wrapper = mountWithEnvironment(Index);
+
+  await Vue.nextTick();
+
+  const listItems = wrapper.findAll('.productions .list-group-item');
+  expect(listItems).toHaveLength(2);
+});
+
+test('returns a found item when searching with an existing production name', async () => {
+  const wrapper = mountWithEnvironment(Index);
+
+  const searchInput = wrapper.find('#productions-overview-search');
```

Comment:
Is there no way to find an input element, checkbox, etc by its label text with the setup we have now? Or if we add another package? I think I remember that was one point we all seemed to agree on, that we wanted to avoid having DOM selectors.

Reply:
By the way, this package is a lightweight adapter built on top of DOM Testing Library (which we originally meant to use) and @vue/test-utils (which we use now if I'm not mistaken). That might be a solution to have the best of both worlds? https://github.com/testing-library/vue-testing-library

An example in their README:

![Screenshot 2020-09-22 at 17 55 12](https://user-images.githubusercontent.com/959026/93906691-caa52480-fcfc-11ea-90bb-d5d4be460bc7.png)

Reply:
We tried that one, but it wasn't clear how to mock the next specific helpers (API, i18n) when rendering